### DISCUSSION TITLE
Game wont launch with BepinEX

### DISCUSSION THREAD

**Discussion by miyonfaga** [BepInEx/BepInEx#772] [Help and Q&A]
I use R2Modman and after installing the sharp modpack the game keeps refusing to launch in the modded option, still launches in vanilla just not modded, even creating a blank profile still gives me the issue.

**Comment by kohramach** [comment]
I use R2Modman and after installing the sharp modpack the game keeps refusing to launch in the modded option, still launches in vanilla just not modded, even creating a blank profile still gives me the issue https://omegle.onl/ .

I got this,...
